src/engine/ruleset/integrations/windows/decoders/windows-sysmon.yml

```diff
@@ -117,41 +280,41 @@ normalize:
     - file.pe.product: $windows.EventData.Product
 
   # DNS event
- - check: $event.code=="22"
+ - check: $event.code == "22"
    map:
     - network.protocol: dns
 
- - check: $windows.EventData.SignatureStatus==Valid
+ - check: $windows.EventData.SignatureStatus == Valid
    map:
     - file.code_signature.valid: True
 
- - check: $windows.EventData.Initiated==True
+ - check: $windows.EventData.Initiated == True
    map:
     - network.direction: egress
 
- - check: $windows.EventData.Initiated==false
+ - check: $windows.EventData.Initiated == false
    map:
     - network.direction: ingress
 
- - check: $windows.EventData.SourceIsIpv6==true
+ - check: $windows.EventData.SourceIsIpv6 == true
    map:
     - network.type: ipv6
 
- - check: $windows.EventData.SourceIsIpv6==false
+ - check: $windows.EventData.SourceIsIpv6 == false
```

Comment:
Should double quotes be added to the literals?

Reply:
They are already string, it is not necessary